templates/emails/failed-refund-customer.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Output the email header.
+ */
+do_action( 'woocommerce_email_header', $email_heading, $email ); ?>
+
+<p>
+	<?php
+
+	echo esc_html(
+		sprintf(
+			// translators: 1) an order number, 2) the reason for the refund request failure.
+			_x(
+				'The refund for order %1$s. Reason:  %2$s.',
```

Comment:
I think we need to include "has failed".
```suggestion
				'The refund for order %1$s has failed. Reason:  %2$s.',
```